packages/database/src/DatabaseChangeChannel.js

```diff
@@ -34,7 +34,8 @@ export class DatabaseChangeChannel extends PGPubSub {
         type: 'update',
         record_type: recordType,
         handler_key: specificHandlerKey,
-        record,
+        old_record: record,
+        new_record: record,
```

Comment:
Yeah, it is a bit weird, but it does serve the practical purpose of saying "mark these records as needing to be handled as though they were updated, even though nothing has changed, because some dependency _has_ been updated, and that needs to be acknowledged by the system". E.g., when a survey response's date changes, we publish changes for all the answers inside, even though nothing about the answer itself has changed.

Because this is pretending to be an 'update', it does need both old_record and new_record to be present, and it's valid that they're both the same

Reply:
Sounds good, thanks for the explanation!